src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/fileinput/_cq_template.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We also need to create proxies for forms and communication portal component in archetype. Name should be 

`${appTitle} - Forms and Communications Portal`